src/seedu/addressbook/AddressBook.java

```diff
@@ -159,6 +159,20 @@
      * If the first non-whitespace character in a user's input line is this, that line will be ignored.
      */
     private static final char INPUT_COMMENT_MARKER = '#';
+    
+    /**
+     * length of input in processProgramArgs. If length of input is greater than 2, then it is invalid;
+     * if length of input is 1, then setup a given file for storage
+     * if length of input is 0, then setup a default file
+     */
+    private static final int LENGTHINVALID = 2;
+    private static final int LENGTHSETUPGIVENFILE = 1;
+    private static final int LENGTHSETUPDEFAULT = 0;
```

Comment:
should be "LENGTH_INVALID", "LENGTH_SET_UP_GIVEN_FILE" and "LENGTH_SET_UP_DEFAULT"